engine-tests/src/tests/erc20_connector.rs

```diff
@@ -701,7 +812,47 @@ pub mod sim_tests {
             )
             .assert_success();
 
-        // 3. Deploy NEP-141
+        // 3. Deploy wnear and set wnear address
+        let wnear = crate::tests::xcc::deploy_wnear(&aurora);
+        let wnear_erc20 = deploy_erc20_from_nep_141(&wnear, &aurora);
+        aurora
+            .call(
+                "factory_set_wnear_address",
+                wnear_erc20.0.address.as_bytes(),
+            )
+            .assert_success();
+
+        // 4. Transfer wnear to `ft_owner` and bridge it to aurora
+        transfer_nep_141(
+            &wnear,
+            &aurora.user,
+            ft_owner.account_id().as_str(),
+            FT_TOTAL_SUPPLY,
+        );
+
+        transfer_nep_141_to_erc_20(
+            &wnear,
+            &wnear_erc20,
+            &ft_owner,
+            ft_owner_address,
+            FT_TRANSFER_AMOUNT,
+            &aurora,
+        );
+
+        assert_eq!(
+            nep_141_balance_of(ft_owner.account_id.as_str(), &wnear, &aurora),
+            FT_TOTAL_SUPPLY - FT_TRANSFER_AMOUNT
+        );
+        assert_eq!(
+            nep_141_balance_of(aurora.contract.account_id.as_str(), &wnear, &aurora),
+            FT_TRANSFER_AMOUNT
+        );
+        assert_eq!(
+            erc20_balance(&wnear_erc20, ft_owner_address, &aurora),
+            FT_TRANSFER_AMOUNT.into()
+        );
+
+        // 5. Deploy NEP-141
```

Comment:
Is this just some random token (so not wNEAR)? What is the purpose of it? Most of the steps look the same to the above wNEAR deployment so I'm not sure what this means.

Reply:
The `wNEAR` is more than just nep141 token, it is a native near custodian, so it has extra api/methods.
Anyway, it is a generic context for the tests, so we need to deploy both of them to cover the nep141 and wNEAR tokens behaviors.